cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -53,6 +53,26 @@ EOF
   echo "will deprovision GCE cluster ${infraID} in region ${region}"
 done
 
+libvirt_p_cluster_age_cutoff="4 hours ago"
+echo "deprovisioning clusters with creation time before ${libvirt_p_cluster_age_cutoff} in libvirt-p"
+for network in $( ssh "${bastion_p}" virsh net-list --all --name | grep -v default ) ; do
+	if $( ssh "${bastion_p}" journalctl -u libvirtd.service --until "${libvirt_p_cluster_age_cutoff}" 2>/dev/null | grep "${network}" ) ; then
```

Comment:
If I am reading that line correctly, the journal log will continue growing until it gets rotated via settings in /etc/systemd/journald.conf 
I hope those are configured so that a rotation won't occur in the middle of a CI run.